libkpod/exec.go

```diff
@@ -0,0 +1,29 @@
+package libkpod
```

Comment:
No more libkpod

Reply:
@rhatdan we have to use libkpod until libpod is ready for container related stuff, which it isn't.  I talked to @mheon about this and we considered progress on the commands to be more important ATM.

Reply:
Especially considering how minimal the backend code on this is. We need to support additional functionality (like attaching to exec'd processes) at some later point, but this gets us some form of exec with almost no code on the backend.

Reply:
Ok I give in, but we really need to start moving to libpod.